### PR TITLE
data: de-maruader profile

### PR DESCRIPTION
**Pull request by de-marauder**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/de-marauder.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name":"Ezike Obiajulu (de-marauder)",
+  "displayStatsPublic": true,
+  "type":"personal",
```
Comment (SaraJaoude) [inline data/de-marauder.json:4]: these are no longer necessary:
```suggestion
```

File: data/de-marauder.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name":"Ezike Obiajulu (de-marauder)",
+  "bio":"Software and DevOps Engineer exploring the web seas",
+  "avatar":"https://github.com/de-marauder.png",
```
Comment (SaraJaoude) [inline data/de-marauder.json:4]: this is not required:
```suggestion
```

File: data/de-marauder.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    {
+      "name": "GitHub",
+      "url": "https://github.com/de-marauder",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Portfolio",
+      "url": "https://de-marauder.vercel.app",
+      "icon": "FaWeb"
+    },
```
Comment (SaraJaoude) [inline data/de-marauder.json:24]: removing comma:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
I have made the necessary changes for the Json to pass so I can now merge this PR

**Comment by de-marauder** [comment]
Thank you @SaraJaoude. One test still seems to be failing. What can I do to fix it?

**Pull request merged**